events/wc_frostwolf_events.txt

```diff
@@ -58,11 +89,11 @@ narrative_event = {
 		}
 	}
 	option = {
-		name = EVTOPTB_WCFW.1
-		custom_tooltip = { text = EVTDESB_WCFW.1 }
+		name = We will be heading south.
+		custom_tooltip = { text = EVTDESB_WCFW_2 }
 		prestige = -10
-		#We have to go south...
-		#Perhaps, the road is going to be challenging, but the Horde has no plans at south.
+		# We have to go to the south...
+		# The road is going to be challenging, but we can establish our home far from the threat.
 		73 = {
 			province_event = {
 				id = WCFW.120
```

Comment:
It says we're goint south, but `WCFW.120` leads to `WCFW.20` (#Way to Dragon Isles).

Reply:
Changed this to point to `WCFW.140` which is the correct start of the "Go South" chain.